packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/sections-three-up~compact.json

```diff
@@ -0,0 +1,53 @@
+{
+  "sectionThreeUp" : {
+    "compact": "True",
```

Comment:
```suggestion
    "compact": "True",
```
I think we should use boolean `true` instead of a string for the value
Please update all the boolean values in this PR

Reply:
changed this value